### DISCUSSION TITLE
Volume Correction in Peridigm

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#271] [Q&A]
Hi everyone,
     I would like to know whether Peridigm includes any volume correction for LPS model, if it is, which method does it use? Thank you for your kind reply.

**Comment by ppandit95** [comment]
Hi @guangdongzhang ,

Can you elaborate on which method as from what I know the method to calculate volume correction factor has been elaborated in Peridynamic Theory and Applications book and I guess the same method has been used in peridigm but in case if there is another method to do so then I certainly would like to know about .

Hope this helps.

Regards
Pushkar
